cmd/aerial/pvfm.go

```diff
@@ -47,16 +47,18 @@ func np(s *discordgo.Session, m *discordgo.Message, parv []string) error {
 
 		return nil
 	} else {
+		result = append(result, ":radio: *Now Playing on PVFM*\n")
```

Comment:
Some testing on my end says that the corresponding emoji are U+1F3B5 MUSICAL NOTE and U+1F4FB RADIO. Does the API support converting to Discord emotes?